shared/wallets/transaction-details/index.tsx

```diff
@@ -301,19 +310,25 @@ const TransactionDetails = (props: NotLoadingProps) => {
           unread={false}
           yourRole={props.yourRole}
           issuerDescription={props.issuerDescription}
+          isAdvanced={props.isAdvanced}
+          summaryAdvanced={props.summaryAdvanced}
         />
       </Kb.Box2>
       <Kb.Divider />
       <Kb.Box2 direction="vertical" gap="small" fullWidth={true} style={styles.container}>
-        <Kb.Box2 direction="vertical" gap="xtiny" fullWidth={true}>
-          <Kb.Text type="BodySmallSemibold">Sender:</Kb.Text>
-          {sender}
-        </Kb.Box2>
+        {sender && (
+          <Kb.Box2 direction="vertical" gap="xtiny" fullWidth={true}>
```

Comment:
Sometimes `{!!sender && (` is necessary to avoid a crash on Android, where the failing && returns a '' that's treated as bare text, that might be there case here too.

Reply:
Thanks! Updated but github doesn't seem to understand.